scheduler/src/cook/passport.clj

```diff
@@ -0,0 +1,12 @@
+(ns cook.passport
+  (:require [clojure.data.json :as json]
+            [clojure.tools.logging :as log]
+            [cook.config :as config]))
+
+(defn log-passport-event
+  [log-data]
+  (log/log config/passport-logger-ns :info nil (json/write-str log-data)))
+
+(def api-job-submission "api-job-submission")
+(def pod-launching "pod-launching")
+(def pod-completed "pod-completed")
```

Comment:
Thoughts on these values being :symbols instead of "strings"?

Reply:
I can change them to :symbols (just tested and the json log code formats them correctly).  These fields are only used as the values of maps and not as keys though-- is it still our convention to have them be :symbols?

Reply:
I'm fine with either solution, I just didn't know that having :symbols as values was preferred.

Reply:
I think it's slightly more idiomatic Clojure to have them as symbols